modules/serverless-enabling-tls-local-services.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * /serverless/security/serverless-config-tls.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-enabling-tls-local-services_{context}"]
+= Enabling TLS authentication for cluster local services
+
+For cluster local services, the Kourier local gateway `kourier-internal` is used. If you want to use TLS traffic against the Kourier local gateway, you must configure your own server certificates in the local gateway.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have administrator permissions.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Deploy server certificates in the `knative-serving-ingress` namespace:
++
+[source,terminal]
+----
+$ export san="knative"
+----
++
+[NOTE]
+====
+These certificates must serve the request against `<app_name>.<namespace>.svc.cluster.local`, so using Subject Alternative Name (SAN) validation is recommended.
```

Comment:
I'd probably change the word "recommended" to "required". I don't see how it can work without the SAN validation because the services might be created in different namespaces and have different names.